Neos.ContentRepository.LegacyNodeMigration/Classes/Processors/EventExportProcessor.php

```diff
@@ -132,14 +102,10 @@ public function run(): ProcessorResult
                 $this->exportEvent(new RootNodeAggregateWithNodeWasCreated($this->workspaceName, $this->contentStreamId, $sitesNodeAggregateId, $this->sitesNodeTypeName, $this->interDimensionalVariationGraph->getDimensionSpacePoints(), NodeAggregateClassification::CLASSIFICATION_ROOT));
                 continue;
             }
-            if ($this->metaDataExported === false && $nodeDataRow['parentpath'] === '/sites') {
-                $this->exportMetaData($nodeDataRow);
-                $this->metaDataExported = true;
-            }
             try {
-                $this->processNodeData($nodeDataRow);
-            } catch (MigrationException $exception) {
-                return ProcessorResult::error($exception->getMessage());
+                $this->processNodeData($context, $nodeDataRow);
+            } catch (MigrationException $e) {
+                throw new \RuntimeException($e->getMessage(), 1729506899, $e);
             }
```

Comment:
that makes no sense here and probably in other places? The `MigrationException` might as well just bubble up.